__tests__/app.test.js

```diff
@@ -0,0 +1,193 @@
+const { getTopics } = require("../db/helpers/utils");
+
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("getTopics", () => {
+  test("Responds with an array of topic objects, slug & description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toEqual([
+          {
+            description: "The man, the Mitch, the legend",
+            slug: "mitch",
+          },
+          {
+            description: "Not dogs",
+            slug: "cats",
+          },
+          {
+            description: "what books are made of",
+            slug: "paper",
+          },
+        ]);
+      });
+  });
```

Comment:
you can test this in a more programmatic way, by confirming the length of the array and the keys on each element in the array. forEach will be handy here, as well as `expect.any(String)` (or other data type)